MuhanguziDavid/mydiary-frontend
# 162233319 setup circleci

### What does this PR do?
Enables reporting of test coverage to codeclimate.

#### Description of Task to be completed?
Currently, there is no way of seeing the quality of the code written.
This PR adds code-climate maintainability and test-coverage badges to show the code quality.

#### How should this be manually tested?
Go to the `ch-setup-circleci-162233319` branch on this github repository and the badges should be displayed in the README.

#### Background context?
NA

#### What are the relevant pivotal tracker stories?
[162233319](https://www.pivotaltracker.com/story/show/162233319)
